server/src/test/java/com/objectcomputing/checkins/services/checkinnotes/CheckinNoteControllerTest.java

```diff
@@ -1,81 +1,67 @@
 package com.objectcomputing.checkins.services.checkinnotes;
 
```

Comment:
Can you run test coverage on this one file and check the code coverage.

Can you add more tests such as 

- creating an already existing checkin note (checkInId and memberProfileId match).
- trying to create a checkin note with a non existing checkin
- trying to create a checkin note with a non existing memberProfile
- A find with expected multiple results return along with a few notes created that won't be chosen, which should not be expected to be in the results.

Reply:
Code looks good.

Reply:
Sure, I can add more tests. I just converted existing tests to use test containers as we have service tests(mock) as well which will take care of other scenarios.

Reply:
Good point...

@mkimberlin Did we decide on whether to abandon the unit tests for impl, and focus all on testcontainers?

Reply:
done. added more tests and it should cover everything.